stock_picking_invoice_product_group/models/account_invoice.py

```diff
@@ -0,0 +1,21 @@
+# -*- coding: utf-8 -*-
+# (c) 2015 Alex Comba - Agile Business Group
+# License AGPL-3 - See http://www.gnu.org/licenses/agpl-3.0.html
+
+from openerp import models, fields
+
+
+class AccountInvoice(models.Model):
+    _inherit = 'account.invoice'
+
+    related_picking_ids = fields.Many2many(
```

Comment:
No answers :sweat_smile:

@pedrobaeza coming back to this case, I think the odoo rule "do not change database structure for stable addons" is generally a good rule and should be also valid for OCA addons, when possible.

The change you propose should be postponed to 9.0 version of the modules